packages/api/src/routes/webhook/fitbit.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import { asyncHandler } from "../util";
+import { processData } from "../../command/webhook/fitbit";
+import { getEnvVar } from "../../shared/config";
+
+const routes = Router();
+
+/** ---------------------------------------------------------------------------
+ * GET /webhook/fitbit
+ *
+ * Verify Fitbit WH subscriber. Receives a verification notification and sends back a verification post request.
+ *
+ */
+routes.get(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    if (req.query.verify) {
+      console.log(
+        "Verifying the subscriber. Make sure to use the verification code provided in the dev.fitbit dashboard!"
+      );
+
+      if (req.query.verify === getEnvVar("FITBIT_SUBSCRIBER_VERIFICATION_CODE")) {
```

Comment:
this should probably use `getEnvVarOrFail()`, and in any case, let's create a getter in the config as per the existing pattern: https://github.com/metriport/metriport/blob/cce5d7eaa9e91aaa4b4bcab885a4ab0997ca3715/packages/api/src/shared/config.ts#L190-L192